docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,52 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled
```

Comment:
Maybe mention the config property to be set?

Reply:
Also, title case the bullet list?

Reply:
I added the config property but didn't title case the bullet list to be consistent with the casing style used in other bullet lists in this section of the documentation.